src/algorithms/motion_tracking/MotionTrackingWrapper.py

```diff
@@ -2,23 +2,41 @@
 import logging
 from datetime import timedelta
 from collections import defaultdict
-from typing import Any, Dict
+from typing import Any, Dict, Tuple
 import numpy as np
-# import matplotlib.pyplot as plt
 
 from algorithms.BaseAgent import BaseAgent
 from utils.RlGlue.agent import AsyncAgentWrapper
 from utils.metrics import UnbiasedExponentialMovingAverage as uema
-# from utils.checkpoint import checkpointable
+from utils.checkpoint import checkpointable
+from PyExpUtils.collection.Collector import Collector
 
 logger = logging.getLogger("plant_rl.MotionTrackingWrapper")
 logger.setLevel(logging.DEBUG)
 
 
-# @checkpointable(("sensitivity", "mean_areas", "openness_record", "openness_trace"))
+@checkpointable(
+    (
+        "target_intensity",
+        "is_first_day",
+        "agent_started",
+        "sensitivity",
+        "mean_areas",
+        "openness_record",
+        "openness_trace",
+    )
+)
 class MotionTrackingWrapper(AsyncAgentWrapper):
-    def __init__(self, agent: BaseAgent):
-        super().__init__(agent)
+    def __init__(
+        self,
+        observations: Tuple[int, ...],
+        actions: int,
+        params: Dict,
+        collector: Collector,
+        seed: int,
+        agent: BaseAgent,
+    ):
```

Comment:
Can you updated everything else that that inherits from AsyncAgentWrapper to use this new interface?